### PR TITLE
Volvo pr Ciel day 6

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 1]
```diff
@@ -279,7 +279,7 @@ Chasing that woman will only get you killed. You mustn't meet Arcueid Brunestud
 [sha:776dcd98f303f69bb408d0a3bbc1b0194cc87027]{
 -- Page 58, Offset 29660.
 -- 「一度でも吸血衝動に負けた真祖は、もう手遅れだからです。
-"If a True Ancestor succumbs to their vampiric urges even once,
+"If a True Ancestor succumbs to their Vampiric Urges even once,
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:282]: did we decide to make this a proper noun? I thought we decided on the opposite

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 2]
```diff
@@ -589,15 +589,15 @@ Senpai was acting as though everything was fine, but her wounds must have been f
...
+Hmm, but you haven't regained your strength yet, have you?
  Alright, how about I give you an IV drip later?
  I already have a stock of IV bags set aside for you, you know?
  If I don't hurry up and use them, Matriarch-chan won't give me any pocket money for the next batch.
 }
 [sha:d1a34b3613a6aca7698f0b83aace470c85e80669]{
 -- Page 158, Offset 13568. Mods: @n.
 -- そんなワケで食後は部屋でウルトラ点滴タイムけってーい！　一時間ぐらいかかるから午前中はあけておいてネ！
-That settles it, after breakfast it's time for some Ultra IV Time in your room!
+That settles it, after breakfast, it's time for some Ultra IV Time in your room!
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:600]: it. After

File: script/Ciel/Day 6/06_01_CIEL06_5.txt [hunk 3]
```diff
@@ -186,12 +186,12 @@ When I give my compliments to the chef, she replies―――
...
 -- 今夜の食材は産地直送、畏れ多くも天皇陛下にもご献上されている逸品ですもの」
 Tonight's ingredients have been sourced fresh from the farm,
- and are considered such delicacies that they've even been served to His Majesty the Emperor."
+ and are considered such delicacies that they've even been served to His Majesty, the Emperor."
 }
 [sha:808d5dd43a2821ed22abcde2f8b04bc68ebbc4ef]{
 -- Page 217, Offset 13716.
 -- なるほど。やはりお高い椎茸さまだったか。
-I see. I didn't realize these were such high-falutin fungi.
+I see. I didn't realize these were such highfalutin fungi.
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_5.txt:194]: I see time crack open before me: a way to the future, death unto our world and all its woes, the shape of things to come. I see... thousands of tweeted screenshots... all of "highfalutin fungi"

> やはりお高い椎茸さまだったか
I didn't realize they were so fancy
I didn't realize they were so upscale.
I didn't realize they were such special shiitake

File: script/Ciel/Day 6/06_01_CIEL06_7.txt [hunk 4]
```diff
@@ -223,14 +223,14 @@ His expression is more than simply anguished. It's closer to hatred.
...
- But this time around I didn't want to waste even a single second.
+ But this time around, I didn't want to waste even a single second.
  Roa isn't worth my time."
 }
 [sha:50658100157dab4b2b1b8764be203ed601ee980c]{
 -- Page 268, Offset 13864.
 -- 「ただの気まぐれかよ。なんだそれ。何か悪いモンでも食べたのか？　それとも故障が近いのか？　精密機械が気分でパターン変えてんじゃねえよ、クソ！」
 "You did this on a whim?!
- Fuck off.
+ Fuck off!
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_7.txt:233]: is this change based on the va? from the text it looks like a curt "fuck off."

Unless the va is quite angry, I'd revert

File: script/Ciel/Day 6/06_01_CIEL06_7.txt [hunk 5]
```diff
@@ -302,9 +302,10 @@ Despite being on the receiving end of his murderous gaze,
 -- Page 273, Offset 13875.
 -- 「まあいい、終わった事をウダウダ言っても時間の無駄だ。人生残り少ねぇんだし前向きにいかねえとな。
 // This guy _is_ Laurentis, right? 17 > 12 is I assume the problem here, and so that 人生残り is his own
+// spilled or spilt?
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_7.txt:305]: spilt in america but i wouldn't look twice at spilled

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Arcueid/Day 11/11_00_ARC11_2.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 11/11_00_ARC11_2.txt:282]: Petrikow marked it caps on the glossary, I guess take it up with him
https://tsukihimates.com/glossary/#vampires_vampiric_urge

File: script/Ciel/Day 6/06_01_CIEL06_7.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Ciel/Day 6/06_01_CIEL06_7.txt:233]: Yeah this is a 'don't be ridiculous' not an imperative

File: script/Ciel/Day 6/06_01_CIEL06_7.txt [hunk 6]
```diff
@@ -330,17 +331,17 @@ Everyone can go pat themselves on the back for resolving the incident in 'this c
...
 -- Page 274, Offset 13880.
 -- ……けどなあ、お姫さま。アンタはいいのかよ、それで」
-Still, princess... You sure you're doing okay?"
+Still, Princess... You sure you're doing okay?"
 }
 [sha:7a0bd8125aa6e2ce19f54628eddd9d95d4101763]{
 -- Page 275, Offset 13881.
 -- 「へえ―――いいって、何が？」
-"Oh? Okay how?"
+"Oh? Okay, how?"
```
Comment (rschlaikjer) [inline script/Ciel/Day 6/06_01_CIEL06_7.txt:344]: I'd revert - "Okay how?" to me is "what is it that makes you ask I'm okay?" whereas "okay, how" is more "okay I acknowledge a thing you've just said, how [asking new question]?"

**Review by rschlaikjer — APPROVED** [review]

File: script/Ciel/Day 6/06_01_CIEL06_5.txt (hunk 3 above)
Comment (yk1512) [reply script/Ciel/Day 6/06_01_CIEL06_5.txt:194]: jesus

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 6/06_01_CIEL06_7.txt (hunk 4 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 6/06_01_CIEL06_7.txt:233]: Yeah it was. I double checked and I change my mind. The exclamation point doesn't fit that well

File: script/Ciel/Day 6/06_01_CIEL06_7.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 6/06_01_CIEL06_7.txt:305]: spilt it is then

File: script/Arcueid/Day 11/11_00_ARC11_2.txt (hunk 1 above)
Comment (Volvo-hackangel) [reply script/Arcueid/Day 11/11_00_ARC11_2.txt:282]: yeah it is in the glossary but not in the spreadsheet so I'm unsure what to do with this really
image

**Pull request merged**